New package: ExportUtils v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: ExportUtils
- Repository: https://github.com/BrianDouglasIE/ExportUtils.jl
- Created by: @BrianDouglasIE
- Version: v0.1.0
- Commit: 9c4ed521e15d60c424b603d7777d4256dc91142f
- Git reference: HEAD
- Description: A package providing utilities to simplify Julia module exports.

goerz commented:
[noblock] You have to retrigger the registration and edit your comment to contain `[noblock]`

JuliaHubRegistrator committed:
New package: ExportUtils v0.1.0

UUID: f05fbe7a-e913-40c7-b48b-8529e8cb1884
Repo: https://github.com/BrianDouglasIE/ExportUtils.jl.git
Tree: 711a6347d23a9c8c9b448decb872079f99a254d2

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8

Seelengrab commented:
This package IMO falls into the "personal utility package" category:

> The General registry is not a place for "personal packages" that consist of collections of "utility functions"

goerz commented:
[noblock] This seems totally fine to me. It was the result of a discussion at https://julialang.zulipchat.com/#narrow/stream/137791-general/topic/Export.20everything.20from.20a.20module/near/465268373, so it already has the interest of more than the original author.

goerz commented:
[noblock] A thread you were a part of, I just realized. So, you really find it objectionable to have this solution packaged up?

Seelengrab commented:
I was only part of the initial question, not of the follow up discussion. Still, in the followup discussion there was an [existing package](https://github.com/dalum/InlineExports.jl) pointed out which already does this (and is also registered), so I'm mostly questioning what this package does so differently that it couldn't be done with the existing one.

goerz commented:
[noblock] It’s a valid point. So @BrianDouglasIE should make an argument for the package and try to convince @Seelengrab that this should be merged. Or, if no consensus can be reached, find a registry maintainer on Slack and convince them to merge manually
